### PR TITLE
Pre-generated SRS loading 

### PR DESCRIPTION
**Pull request by ToufeeqP**
Companion PRs:
avail-core [PR#58](https://github.com/availproject/avail-core/pull/58)
avail [PR#270](https://github.com/availproject/avail/pull/270)

### CODE REVIEW

File: src/bin/avail-light.rs [hunk 1]
```diff
@@ -192,7 +192,7 @@ async fn run(error_sender: Sender<anyhow::Error>) -> Result<()> {
 	#[cfg(feature = "network-analysis")]
 	tokio::task::spawn(network_analyzer::start_traffic_analyzer(cfg.port, 10));
 
-	let pp = Arc::new(kate_recovery::testnet::public_params(1024));
+	let pp = Arc::new(kate_recovery::testnet_v2::public_params());
```
Comment (prabal-banerjee) [inline src/bin/avail-light.rs:195]: Let's use the Couscous?

File: Cargo.lock [hunk 2]
```diff
@@ -442,13 +442,13 @@ dependencies = [
  "parity-scale-codec",
  "scale-info",
  "serde",
- "sp-arithmetic 16.0.0",
- "sp-core 21.0.0",
- "sp-io 23.0.0",
+ "sp-arithmetic 7.0.0",
```
Comment (aterentic-ethernal) [inline Cargo.lock:445]: I think we should revert changes which reduces versions to old ones, it seems that every update switches to old/new versions randomly.

**Review by aterentic-ethernal — APPROVED** [review]
Also, I think we should update public parameters in test, but not 100% sure.

File: Cargo.lock (hunk 2 above)
Comment (ToufeeqP) [reply Cargo.lock:445]: Ah, I see, No idea why this is happening. but will revert it

**Pull request merged**